Assets/Scripts/ROS Scripts/ROS Connections v2/Drones/Matrice_ROSDroneConnection.cs

```diff
@@ -511,6 +566,29 @@ public ROSBridgeMsg OnReceiveMessage(string topic, JSONNode raw_msg, ROSBridgeMs
                 local_position = new Vector3(pointMsg.GetX(), pointMsg.GetY(), pointMsg.GetZ());
                 result = pointMsg;
                 Debug.Log(result);
+                break;
+            case "dji_sdk/rtk_position":
+                gps_position = (parsed == null) ? new NavSatFixMsg(raw_msg) : (NavSatFixMsg)parsed;
```

Comment:
you can probably put the rtk_position and gps position code in one method and call that method instead. that way we can change code easily.

Reply:
good call, done